<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetreiveCloudWgt</class>
 <widget class="QWidget" name="RetreiveCloudWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloud Blockpads</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
    border: 2px solid grey;
    border-radius: 10px;
    margin-top: 6px;
    padding-top: 8px;
}

QGroupBox::title {
    margin: 0 5px;
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
}

QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
	min-width: 100px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QTableWidget {
    border: 2px solid black;
    border-radius: 10px;
    padding: 2px;
}

QHeaderView{
    background-color:white;
}
QHeaderView::section{
	border-top:0px solid #D8D8D8;
    border-left:0px solid #D8D8D8;
    border-right:1px solid #D8D8D8;
    border-bottom: 1px solid #D8D8D8;
    border-radius: 10px;
    background-color:#f0f0f0;
    padding:4px;
}
        QTableCornerButton::section{
            border-top:0px solid #D8D8D8;
            border-left:0px solid #D8D8D8;
            border-right:1px solid #D8D8D8;
            border-bottom: 1px solid #D8D8D8;
    		border-radius: 10px;
            background-color:white;
        }

QLabel{
	min-width:200px;
	max-width:200px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxFxbotAccount">
     <property name="title">
      <string>TotalCryptos.com account</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelUsername">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditUsername"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelPassword">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLogin">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBlockPads">
     <property name="title">
      <string>BlockPads</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidgetCloudBlockPads">
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelDownloadFolder">
          <property name="text">
           <string>Download Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDownloadFolder"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="wgtOpenAfter" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelAutoOpenBlockPad">
           <property name="text">
            <string>Open After Download</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAutoOpenBlockPad">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
